operator/charts/kit-operator/values.yaml

```diff
@@ -8,13 +8,25 @@ controller:
   env: []
   nodeSelector: {}
   tolerations: []
-  affinity: {}
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: karpenter.sh/provisioner-name
+                operator: DoesNotExist
   # TODO this will be updated by the git actions
   image: "public.ecr.aws/kit/kit-operator:latest"
 webhook:
   env: []
   nodeSelector: {}
   tolerations: []
-  affinity: {}
+  affinity:
```

Comment:
Can we look into mirroring karpenter's helm chart? I don't see a reason for them to be too different.

Reply:
Yes I totally agree. The value structure is trying to mimic what Karenter has, except now we still separate controller and webhook to separate pods. Will need to merge these two pods to have the exact same structure.

Let's work on it later.